.circleci/windows/assemble.bat

```diff
@@ -0,0 +1,57 @@
+@echo off
+REM
+REM GRAKN.AI - THE KNOWLEDGE GRAPH
+REM Copyright (C) 2019 Grakn Labs Ltd
+REM
+REM This program is free software: you can redistribute it and/or modify
+REM it under the terms of the GNU Affero General Public License as
+REM published by the Free Software Foundation, either version 3 of the
+REM License, or (at your option) any later version.
+REM
+REM This program is distributed in the hope that it will be useful,
+REM but WITHOUT ANY WARRANTY; without even the implied warranty of
+REM MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+REM GNU Affero General Public License for more details.
+REM
+REM You should have received a copy of the GNU Affero General Public License
+REM along with this program.  If not, see <https://www.gnu.org/licenses/>.
+REM
+
+REM needs to be called such that software installed
+REM by Chocolatey in prepare.bat is accessible
+CALL refreshenv
+
+REM build grakn-core-all-windows archive
+bazel build @graknlabs_grakn_core//:assemble-windows-zip || GOTO :error
+
+REM unpack and start Grakn server
+unzip bazel-genfiles\external\graknlabs_grakn_core\grakn-core-all-windows.zip -d bazel-genfiles\dist\ || GOTO :error
+PUSHD bazel-genfiles\dist\grakn-core-all-windows\
+CALL grakn.bat server start || GOTO :error
+CALL grakn.bat console -f ..\..\..\test\helpers\gene.gql -k gene || GOTO :error
```

Comment:
@vmax AFAIR, the `gene` dataset is only required for the `e-2-e` tests and the loading of the dataset into the keyspace is actually taken care within the test setups